src/sage/modular/arithgroup/farey_symbol.pyx

```diff
@@ -1,4 +1,6 @@
 # distutils: sources = sage/modular/arithgroup/sl2z.cpp sage/modular/arithgroup/farey.cpp
+# distutils: language = c++
+# distutils: extra_compile_args = -std=c++11
```

Comment:
There's no harm in compiling with C++11, but the previous version also worked with C++11; C++17 removed many features deprecated in C++11, hence the compile failures (obviously, Sage can't build with an older STD because of the usage of new features added in C++11).
I can't say the same for all dependencies, but Sage itself should compile fine with `-std=c++17`... just adding this for posterity.

Reply:
On the system that I tested, without any `-std=...`, it uses some pre-11 standard. I had to add the `-std=c++11` to fix it. This is the only directive that is currently in use in the Sage extension modules per `git grep 'std=c++' src/sage`. I agree that we could switch to -std=c++17 throughout, but that's beyond the scope of this PR